nusight2/src/client/components/localisation/view.tsx

```diff
@@ -201,3 +202,19 @@ const FieldLineDots = ({ model }: { model: LocalisationModel }) => (
     )}
   </>
 );
+
+const Ball = ({ model }: { model: LocalisationModel }) => (
```

Comment:
This component renders multiple balls:

```suggestion
const Balls = ({ model }: { model: LocalisationModel }) => (
```